applications/serial_lte_modem/prj.conf

```diff
@@ -57,6 +57,18 @@ CONFIG_DEVICE_POWER_MANAGEMENT=y
 # Enable SUPL client support
 #CONFIG_SUPL_CLIENT_LIB=y
 
+# FOTA
+CONFIG_HTTP_PARSER_URL=y
+CONFIG_FOTA_DOWNLOAD=y
+CONFIG_FOTA_DOWNLOAD_PROGRESS_EVT=y
+CONFIG_DFU_TARGET=y
+CONFIG_DOWNLOAD_CLIENT=y
+CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
+CONFIG_BOOTLOADER_MCUBOOT=y
```

Comment:
Is 4096 necessary for FOTA? Default DOWNLOAD_CLIENT_STACK_SIZE is 1024, range 768 - 4096.

Reply:
config DOWNLOAD_CLIENT_STACK_SIZE
	int "Thread stack size"
	range 768 4096
	default 1024

Following assert_tracker to define max stack size just in case.
There is a sanity check in download_client, must be > 512 + Filename Size + Hostname size